scalameta/semantic/shared/src/main/protobuf/semanticdb.proto

```diff
@@ -1,19 +1,11 @@
 syntax = "proto3";
 
-package scala.meta.internal.semantic.proto;
-
-message Database {
-  // A database is treated as a sequence of attributed source files.
-  // This accommodates incremental compilation, because database entries
-  // map one-to-one to compilation units.
-  // This also makes merging databases trivial, as the bytes of two databases
-  // can be simply concatenated to produce a new database.
-  repeated AttributedSource sources = 1;
-}
+package scala.meta.internal.semantic.schema;
 
-message AttributedSource {
+message Attributes {
   reserved 1;
   string path = 5;
```

Comment:
For #777, this field will be removed. Relative paths are already encoded in `v.Entry.name`, so we don't need to duplicate them in `v.bytes` (i.e. `s.Attributes`).